lms/static/styles/frontend_apps.scss

```diff
@@ -1,4 +1,4 @@
 // CSS Entry point for front-end (preact) app
 @use 'base';
 @use 'components';
-@use 'utilities';
+@use 'tailwindcss/utilities';
```

Comment:
The entire `utilities` layer is just Tailwind's utilities layer. We still add a couple of things to the `base` and `components` layers. There is a SASS limitation that `@use` directives have to come first in a module, so `base` and `components`, while they are very small modules, need to remain external. i.e. we can't have something like this in a single module:

```scss
@use 'tailwindcss/base';

// Element styles
@layer base {
  html,
  body {
    height: 100%;
  }
  // ...
}

@use 'tailwindcss/components`;

@layer components {
  // A warning banner shown at the top of the view in unsupported browsers.
  // This is styled to match warning toasts and banners in the Hypothesis client.
  .browser-check-warning {
    // ...
  }
}

//...

@use 'tailwindcss/utilities';
```

even though, once processed, that's the net result. The layers need to be "activated" in order, and we can't split up `@use` directives.